ndcube/ndcube_sequence.py

```diff
@@ -41,15 +41,17 @@ def __init__(self, data_list, meta=None, common_axis=None, **kwargs):
     @property
     def dimensions(self):
         dimensions = [len(self.data) * u.pix] + list(self.data[0].dimensions)
-        # If there is a common axis, length of cube's along it may not
-        # be the same. Therefore if the lengths are different,
-        # represent them as a tuple of all the values, else as an int.
-        if self._common_axis is not None:
-            common_axis_lengths = [cube.data.shape[self._common_axis] for cube in self.data]
-            if len(np.unique(common_axis_lengths)) != 1:
-                common_axis_dimensions = [cube.dimensions[self._common_axis] for cube in self.data]
-                dimensions[self._common_axis+1] = u.Quantity(
-                    common_axis_dimensions, unit=common_axis_dimensions[0].unit)
+        if len(dimensions) > 1:
```

Comment:
Despite the large diff block, this line is the only change.  The other lines are just indented accordingly.